### PR TITLE
Add Wlv and Zvv config

### PR DESCRIPTION
**Pull request by GLP90**

### CODE REVIEW

**Comment by perrozzi** [comment]
comment: I wonder if the folder with the datasets needs to be a "property" of the user analysis folder or not. my naive answer would be no, I would centralize them somewhere. what do you think?
(not necessarily to be done in the context of this PR)

File: python/submitThem_old.py [hunk 1]
```diff
@@ -607,7 +607,7 @@ def appendList(): optionsList.append({'cut':copy(_cut),'var':copy(_treevar),'nam
         if shape__.replace(' ','')  == '': continue#to avoid && case
         shapecut_split_ = shape__.split('||')
         for shape_ in  shapecut_split_:
-            new_cut_list=sys_cut_suffix[syst]
```
Comment (perrozzi) [inline python/submitThem_old.py]: so we still need to differentiate the submitThem,py and submitThem_old? probably not

Comment (perrozzi) [reply python/submitThem_old.py]: (once more, not necessarily needs to be addressed in this PR. we can probably start using issues to track things to be changed)

Comment (perrozzi) [reply python/submitThem_old.py]: (issues need to be activated in forked repositories, just go to settings and then tick on "Issues"

Comment (GLP90) [reply python/submitThem_old.py]: @perrozzi I agree concerning the dataset folder. We can put it in Xbb/python.

Comment (piberger) [reply python/submitThem_old.py]: @perrozzi I agree too.

File: python/write_regression_systematics.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env pythoelen
```
Comment (piberger) [inline python/write_regression_systematics.py]: what is this?

Comment (GLP90) [reply python/write_regression_systematics.py]: Yes it's a dummy typo

Comment (GLP90) [reply python/write_regression_systematics.py]: (fixed)

**Pull request merged**